internal/selfmonitor/config/config_builder.go

```diff
@@ -37,12 +37,16 @@ func makeScrapeConfig() []ScrapeConfig {
 	return []ScrapeConfig{
 		{
 			JobName: "kubernetes-service-endpoints",
+
 			RelabelConfigs: []RelabelConfig{{
 				SourceLabels: []string{"__meta_kubernetes_service_annotation_prometheus_io_scrape"},
 				Regex:        "true",
 				Action:       Keep,
 			}},
-			KubernetesDiscoveryConfigs: []KubernetesDiscoveryConfig{{Role: RoleEndpoints}},
+			KubernetesDiscoveryConfigs: []KubernetesDiscoveryConfig{{
+				Role:       RoleEndpoints,
+				Namespaces: Names{Name: []string{"kyma-system"}},
```

Comment:
We should not hard-code kyma-system. Instead, we should use the CLI flag/env var with the manager namespace. Please look at `telemetryNamespace` in `main.go`

Reply:
fixed